packages/function-tree/src/FunctionTree.test.js

```diff
@@ -240,4 +240,25 @@ describe('FunctionTree', () => {
     assert(branchStartCount, 2)
     assert(branchEndCount, 2)
   })
+  it('should pass correct input on empty paths', () => {
+    let branchStartCount = 0
+    let branchEndCount = 0
```

Comment:
Lint errors: branchStartCount and branchEndCount are assigned a value but never used.